modules/installing-oadp-aws-sts.adoc

```diff
@@ -0,0 +1,260 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-oadp-aws-sts_{context}"]
+= Installing the OADP Operator and providing the IAM role
+
+AWS Security Token Service (AWS STS) is a global web service that provides short-term credentials for IAM or federated users. This document describes how to install {oadp-first} on an {aws-short} {sts-short} cluster manually.
```

Comment:
Yay attributes! Good job! Might consider going through and seeing where attributes should be used, but aren't.